aspnetcore/grpc/troubleshoot.md

```diff
@@ -168,4 +168,55 @@ You can workaround this issue by:
 
 The WPF application can use the gRPC generated types from the new class library project.
 
+## Calling gRPC services hosted in a sub-directory
+
+The path component of a gRPC channel's address is ignored when making gRPC calls. For example, `GrpcChannel.ForAddress("https://localhost:5001/ignored_path")` won't use `ignored_path` when routing gRPC calls for the service.
+
+The address path is ignored because gRPC has a standardized, prescriptive address structure. A gRPC address combines the package, service and method names: `https://localhost:5001/PackageName.ServiceName/MethodName`.
+
+There are some scenarios when an app needs to include a path with gRPC calls. For example, an ASP.NET Core gRPC app is hosted in an IIS directory and the directory needs to be included in the request. When a path is required it can be added to the gRPC call using the custom `SubdirectoryHandler` specified below:
```

Comment:
```suggestion
There are some scenarios when an app needs to include a path with gRPC calls. For example, when an ASP.NET Core gRPC app is hosted in an IIS directory and the directory needs to be included in the request. When a path is required, it can be added to the gRPC call using the custom `SubdirectoryHandler` specified below:
```